docs/specs/workflow.md

```diff
@@ -318,3 +318,99 @@ And then the frontend will receive a response to telling it to redirect.
 
 Finally the frontend will be visited again with `https://myapp.com/oauth-redirect`.
 The frontend can include necessary information in the redirect URI to make the resumption possible.
+
+### Intent for login id migration (Experimental)
+
+The Intent for login id migration allows the developer to create a workflow for migration.
+
+[![](https://mermaid.ink/img/pako:eNqdUl1rwkAQ_CvLPdv2PQ9Cq4JS_EArfWhK2SYbc2ju0ru9ioj_vZvGaCVgSwMHuZ2ZnZu926vEpqQi5ekjkEmor3HlsIgNyFeiY53oEg3DfeB8RejaSC94tsVy1EaG1q4BPfTJWLiDZ3rPpVLzGtVNt9u0jqDnCJlAGybDYy0nkd3WunW2sdta15BF17SIYE4cnDkxYdTvSBPPKIlkc8Vx5mxClALnZyPYas7hafo4mLQ8q0iiQu9rArCtY35qhHO6qnRi96wcxYWE4eVVp5JM8-72mG5RUgJoUoFQXCowQbbuAs-cLWA2nE4Gb5Pl-GEwvzA5hzmO4X8urah_GVJV8SL2v1_Otfv4SfahoAunil0t1VEFuQJ1Ku91_11VwisoVpH8ppRh2HCsYnMQquS0i51JVCSjp44KZSpBj89bRRluPB2-AAamCc8?type=png)](https://mermaid.live/edit#pako:eNqdUl1rwkAQ_CvLPdv2PQ9Cq4JS_EArfWhK2SYbc2ju0ru9ioj_vZvGaCVgSwMHuZ2ZnZu926vEpqQi5ekjkEmor3HlsIgNyFeiY53oEg3DfeB8RejaSC94tsVy1EaG1q4BPfTJWLiDZ3rPpVLzGtVNt9u0jqDnCJlAGybDYy0nkd3WunW2sdta15BF17SIYE4cnDkxYdTvSBPPKIlkc8Vx5mxClALnZyPYas7hafo4mLQ8q0iiQu9rArCtY35qhHO6qnRi96wcxYWE4eVVp5JM8-72mG5RUgJoUoFQXCowQbbuAs-cLWA2nE4Gb5Pl-GEwvzA5hzmO4X8urah_GVJV8SL2v1_Otfv4SfahoAunil0t1VEFuQJ1Ku91_11VwisoVpH8ppRh2HCsYnMQquS0i51JVCSjp44KZSpBj89bRRluPB2-AAamCc8)
+
+#### Configuration in `authgear.yaml`
+
+```yaml
+migration_workflow:
+  url: authgeardeno:///deno/migration.ts
+  timeout: 10
```

Comment:
I tend to not couple account migration with workflow.

```
account_migration:
  hook:
    url: xxx
    timeout: 10
```

Reply:
How about calling it `login_id_migration` instead of `account_migration`? Not sure if we will have another type of migration that is related to accounts as well.

And the intent is named `IntentMigrateLoginID`

Reply:
Also renamed the intent to `IntentMigrateAccount`